content/en/building/guides/interoperability/openhim.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "OpenHIM Mediators"
+linkTitle: "OpenHIM Mediators"
+weight:
+description: >
+   Guide to running OpenHIM and Mediators with Docker Compose
+keywords: openmrs interoperability
+relatedContent: >
+  building/guides/interoperability/cht_config
+  building/guides/interoperability/ltfu_reference_workflow
+  building/guides/interoperability/troubleshooting
+  building/concepts/interoperability/
+  building/guides/integrations/openmrs/
+---
+
+#### Overview
+
+The components and reference information for interoperability used in this project are:
+
+- [OpenHIE](https://ohie.org/) defines the architecture for an interoperability layer.
+- [OpenHIM](http://openhim.org/) is a middleware component designed to ease interoperability between systems.
+- [HL7 FHIR](https://www.hl7.org/fhir/index.html) is a messaging format to allow all systems to understand the format of the message.
+
+#### CHT
+
+The structure of documents in the CHT database reflect the configuration of the system, and therefore do not map directly to a FHIR message format. To achieve interoperability we used a middleware to convert the CHT data structure into a standardized form so the other systems can read it. Below is the standard data workflow:
+
+![](flow.png)
+
+This project uses OpenHIM as the middleware component with [Mediators](http://openhim.org/docs/configuration/mediators/) to do the conversion. [Outbound Push](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/) is configured to make a request to the middleware when relevant documents are created or modified in the CHT. A Mediator then creates a FHIR resource which is then routed to OpenHIM. OpenHIM routes the resource to any other configured systems.
+
+Conversely, to bring data into the CHT, OpenHIM is configured to route the updated resource to the Mediator, which then calls the relevant [CHT APIs](https://docs.communityhealthtoolkit.org/apps/reference/api/) to update the document in the CHT database. This will then be replicated to users’ devices as per usual.
+
+See more information on the [CHT interoperability](https://docs.communityhealthtoolkit.org/apps/concepts/interoperability/) page on the CHT documentation site.
+
+### Prerequisites
+
+- `docker`
+- `Postman` or similar tool for API testing. This will play the role of the `Requesting System` from the sequence diagram above.
```

Comment:
The diagram linked on line 28 is not the form submission sequence diagram. Do we need the second sentence?